setup.py

```diff
@@ -18,6 +18,7 @@
     # https://github.com/iterative/dvc/issues/8513#issuecomment-1298761683
     "botocore>=1.22.8,<1.28.0",
     "gcsfs>=2021.4.0",
+    "google-cloud-batch>=0.2.0",
```

Comment:
For now, I think I prefer defining this as an extras dependency below.

```py
extras = {
    "glue": ["pandas", "pyarrow", "pyspark"],
    "gbatch": ["google-cloud-batch>=0.2.0"],
    "k8s": "kubernetes>=22.6",
    "viz": "pygraphviz",
}
```

Reply:
SG. Done.